<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgAdminUsuarios</class>
 <widget class="QDialog" name="dlgAdminUsuarios">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Administraci√≥n de Usuarios</string>
  </property>
  <widget class="QLabel" name="lblUsuario">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Usuario:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leUsuario">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="gbAcceso">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Acceso</string>
   </property>
   <widget class="QRadioButton" name="rbOperador">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Operador</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbAdmin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Admin</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pbGuardar">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>150</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbBlanquearPass">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>180</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Blanquear
Password</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="twUsuarios">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>181</width>
     <height>151</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>1</number>
   </property>
   <column>
    <property name="text">
     <string>Usuarios</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pbEliminar">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Eliminar</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>leUsuario</tabstop>
  <tabstop>rbOperador</tabstop>
  <tabstop>rbAdmin</tabstop>
  <tabstop>pbGuardar</tabstop>
  <tabstop>pbBlanquearPass</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
